DashKit/UnfinalizedDose.swift

```diff
@@ -82,9 +82,6 @@ public struct UnfinalizedDose: RawRepresentable, Equatable, CustomStringConverti
         get {
             return duration != nil ? startTime.addingTimeInterval(duration!) : nil
         }
```

Comment:
[nit] I think now that there's no `set`, you can do away with the `get` directive...

```suggestion
        return duration != nil ? startTime.addingTimeInterval(duration!) : nil
```